scripts/plugin_list.py

```diff
@@ -0,0 +1,85 @@
+import datetime
+import pathlib
+import re
+
+import packaging.version
+import requests
+import tabulate
+
+FILE_HEAD = r"""Plugins List
+============
+
+The plugins are listed automatically from PyPI.
+Only PyPI projects that match "pytest-\*" are considered plugins.
+Packages classified as inactive are also excluded.
```

Comment:
It would be interesting to have a count of the number of the plugins in the page, like we have currently with the plugincompat site:

![image](https://user-images.githubusercontent.com/1085180/96750519-68dee580-13a2-11eb-9a3d-4c11d8a7cf00.png)

(Just the plugin count is enough, no need to also show the latest version)

Reply:
I added the plugin count!